ibm_catalog.json

```diff
@@ -269,12 +244,12 @@
               "key": "exclude_cn_from_sans",
               "options": [
```

Comment:
Booleans don't need explicit dropdowns like this - they will have one already. Please remove from all boolean type inputs in this file